apps/common-app/package.json

```diff
@@ -35,7 +35,7 @@
     "react-native-safe-area-context": "5.3.0",
     "react-native-screens": "patch:react-native-screens@npm%3A4.10.0#~/.yarn/patches/react-native-screens-npm-4.10.0-a9c3a4e55e.patch",
     "react-native-svg": "15.12.0",
-    "react-native-worklets": "workspace:*",
+    "react-native-worklets": "0.3.0",
```

Comment:
I guess we should still use `react-native-worklets` from our workspace here, right?

Reply:
Cool, haven't noticed the commit title, only scanned through the PR description. Thanks for explaining!